client/src/components/ReviewAnswers.js

```diff
@@ -1,13 +1,90 @@
 import React from 'react';
-import { Button, Card } from 'react-bootstrap';
+import { Button, Card, Form, Container } from 'react-bootstrap';
 
 const getAnswersClass = (question, userAnswers, answer) => {
   if (answer.isCorrect) {
-    return 'text-success';
+    return 'bg-success text-white';
   } else if (userAnswers[question._id].includes(answer._id)) {
-    return 'text-danger';
+    return 'bg-danger text-white';
   } else {
-    return '';
+    return 'bg-light';
+  }
+};
+
+const getAnswersMark = (question, userAnswers, answer, selectedAnswers) => {
+  if (answer.isCorrect) {
+    if (selectedAnswers.includes(answer._id)) {
+      return (
+        <span
+          style={{
```

Comment:
Most software teams prefer putting CSS in style sheets rather than directly putting it in the `style` attribute, so these CSS properties could be moved to the style sheets